### DISCUSSION TITLE
Push a branch other than the currently-checked-out branch

### DISCUSSION THREAD

**Discussion by canton7** [jesseduffield/lazygit#4088] [Ideas]
Hi,

Is it possible to push a branch other than your currently-checked-out branch? `P` with a branch selected in the branches pane still pushes the checked-out branch, which I found surprising.

Thanks!

**Comment by stefanhaller** [comment]
No, that's not currently possible. Some discussion about this happened in #2755 (and #2795).

**Reply by canton7** [reply]
Aha thanks! For some reason those didn't show up in my searches.
